iree/compiler/Dialect/VM/Target/C/CModuleTarget.cpp

```diff
@@ -299,19 +441,19 @@ LogicalResult translateModuleToC(IREE::VM::ModuleOp moduleOp,
   printInclude("iree/vm/native_module.h");
   printInclude("iree/vm/ref.h");
   printInclude("iree/vm/stack.h");
-  output << "\n";
-
+  printInclude("iree/vm/shims.h");
   printInclude("iree/vm/c_funcs.h");
```

Comment:
Style nit: it's not required but nice to keep the code that's output looking consistent, but if headers are in the same group (no separating vertical whitespace) then they should be in order. The implication here is that it should be valid to include the headers in any order and if you see things out of order it usually indicates a reason that should be documented (like .inl files that must only be included in very specific ways vs. general purpose .h that are meant to be included anywhere). clang-format does this reordering automagically, but for our generated code it'd be nice where possible to just be consistent :)

Reply:
Done :)